cypress/support/step-definitions/simple-color-picker-steps.js

```diff
@@ -69,39 +31,19 @@ Then('Simple Color Picker availableColors on preview is set to {string}', (value
 });
 
 Then('Simple Color Picker {string} element was selected', (index) => {
-  switch (index) {
-    case 'first':
-      simpleColorPickerInput(SECOND_ELEMENT).click();
-      simpleColorPickerInput(FIRST_ELEMENT).click();
-      simpleColorPickerDiv(FIRST_ELEMENT).children()
-        .should('have.attr', 'data-element', 'tick')
-        .should('have.class', 'icon-tick');
-      break;
-    case 'second':
-      simpleColorPickerDiv(SECOND_ELEMENT).children()
-        .should('have.attr', 'data-element', 'tick')
-        .should('have.class', 'icon-tick');
-      break;
-    case 'third':
-      simpleColorPickerDiv(THIRD_ELEMENT).children()
-        .should('have.attr', 'data-element', 'tick')
-        .should('have.class', 'icon-tick');
-      break;
-    default: throw new Error('There is no such index of element');
+  if (index === '1') {
+    simpleColorPickerInput(SECOND_ELEMENT).click();
+    simpleColorPickerInput(FIRST_ELEMENT).click();
+  }
+  for (let i = 1; i < index; i++) {
+    simpleColorPickerDiv(i).children()
+      .should('have.attr', 'data-element', 'tick')
+      .should('have.class', 'icon-tick');
   }
 });
 
-When('I click onto {string} element', (index) => {
-  switch (index) {
-    case 'first':
-      simpleColorPickerInput(FIRST_ELEMENT).click();
-      break;
-    case 'second':
-      simpleColorPickerInput(SECOND_ELEMENT).click();
-      break;
-    case 'third':
-      simpleColorPickerInput(THIRD_ELEMENT).click();
-      break;
-    default: throw new Error('There is no such index of element');
+When('I click onto {int} of element', (index) => {
+  for (let i = 1; i < index + 1; ++i) {
```

Comment:
iterate from 0 to i<index and make +1 here: `simpleColorPickerInput(i+1).click();`

Reply:
fixed